elastic/kibana
# [Obs UX] Apply neutral colours to asset kpi tiles

## Summary

This PR changes the KPI colors to a more neutral color scheme.

| Before | After |
| ------ | ------ |
| <img width="1898" alt="image" src="https://github.com/elastic/kibana/assets/14139027/51bb78e1-cf17-4652-8c77-a3b5e28dbdbe"> | ![image](https://github.com/elastic/kibana/assets/14139027/57a4dc02-4223-44f1-b91a-acf0a9419df9) |

## Testing

- Go to host view:
  - Check the KPI tiles' colors
  - Open the host flyout and check them again
- Go to inventory and click on a host in the waffle map
  -  Check the KPI tiles' colors inside the flyout
  - Click on open as page and check the kpi tiles there:
     - <img width="1658" alt="image" src="https://github.com/elastic/kibana/assets/14139027/a61126ca-9eaf-493e-9d48-bc34ce7843fd">
